client/my-sites/current-site/pending-payment-notice.jsx

```diff
@@ -0,0 +1,36 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import { hasPendingPayment } from 'lib/cart-values';
+
+export const PendingPaymentNotice = ( { translate, cart = {} } ) => {
+	if ( ! hasPendingPayment( cart ) ) {
+		return null;
+	}
+
+	return (
+		<Notice
+			icon="info-outline"
+			isCompact
+			status="is-warning"
+			text={ translate( 'Processing your payment…', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Processing your payment...' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pending-payment-notice)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).